tests/component/sections/create-dataset/CreateDataset.spec.tsx

```diff
@@ -86,6 +93,36 @@ describe('Create Dataset', () => {
         cy.findByText('Not Implemented').should('exist')
       })
   })
+  it('pre-fills the form with user data', () => {
+    cy.customMount(
```

Comment:
I think we have cy.mountAuthenticated custom command for this case?
See [mountAthenticated](https://github.com/IQSS/dataverse-frontend/blob/develop/tests/support/commands.tsx#L62)